python/nav/util.py

```diff
@@ -475,7 +472,7 @@ def auth_token():
     """Generates a hash that can be used as an OAuth API token"""
     from django.conf import settings
 
-    _hash = hashlib.sha1(six.text_type(uuid.uuid4()).encode('utf-8'))
+    _hash = hashlib.sha1(str(uuid.uuid4()).encode('utf-8'))
```

Comment:
SonarCloud flagged this as a potential vulnerability, but it's not your fault. The essence of the code did not change with your PR :)

I posted #2366 to follow up on this.